### DISCUSSION TITLE
Suggestion for an addon

### DISCUSSION THREAD

**Discussion by SpryzenR** [ScratchAddons/ScratchAddons#4208] [Ideas]
This is my first time on github so this is a suggestion for an addon. 

I hate how I when I want to vector everything or bitmap everything I have to do it one by one. So can we add an addon button to make everything vector or bitmap? Thanks.

**Comment by WorldLanguages** [comment]
What do you mean by everything? All costumes inside a single sprite?

**Reply by Secret-chest** [reply]
> What do you mean by everything? All costumes inside a single sprite?

Yes, I think so. Wouldn't make much sense.
(Some think bitmap is pixellated intentionally)

**Reply by flyhigh-1798** [reply]
what if you want a costume to be vector in an all bitmap sprite?

**Reply by SpryzenR** [reply]
> what if you want a costume to be vector in an all bitmap sprite?

scratch already has that feature (vector to bitmap and bitmap to vector)

**Reply by SpryzenR** [reply]
> What do you mean by everything? All costumes inside a single sprite?

Yes all costumes in a single sprite, ig if you wanted to you could make it so it also does all the costume sprites idk.
